spec/controllers/webhooks_controller_spec.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe WebhooksController, type: :controller do
+  subject(:user) { FactoryBot.create(:user) }
+
+  describe 'webhook creation' do
+    before do
+      sign_in(user)
+    end
+
+    context 'with valid params' do
+      let(:valid_request) do
+        post :create, params: { webhook: { topic: 'video/ready', url: 'https://www.tests.com' } },
+                      as: :json
+      end
+
+      it 'returns a 200' do
+        valid_request
+        expect(response).to have_http_status(:created)
+      end
+
+      it 'creates a new webhook' do
+        expect do
+          valid_request
+        end.to change(Webhook, :count).by(1)
+      end
+
+      it "webhook user_id matches current_user's id" do
+        valid_request
+        expect(Webhook.last.user_id).to match(user.id)
+      end
+    end
+
+    context 'with invalid params' do
+      let(:invalid_request) do
+        post :create, params: { webhook: { topic: '', url: '' } }, as: :json
+      end
+
+      it 'returns a 422' do
+        invalid_request
+        expect(response).to have_http_status(:unprocessable_entity)
+      end
+
+      it 'matches an error message' do
+        response = invalid_request
+        body = JSON.parse(response.body)
+        expect(body).to match({ 'errors' => ['Topic can\'t be blank',  'Url can\'t be blank',
+                                             'Url is not a valid URL', 'Topic must be accepted'] })
+      end
+    end
+  end
+
+  describe 'webhook deletion' do
+    context 'when webhook belongs to another user' do
+      subject(:webhook) { FactoryBot.create(:webhook, user_id: user2.id) }
+
+      let(:user2) { FactoryBot.create(:user) }
+      let(:invalid_request) do
+        delete :destroy, params: { id: webhook.public_id }
+      end
+
+      before do
+        sign_in(user)
+      end
+
+      it 'raises ActiveRecord::Record Not Found exception' do
```

Comment:
Use the actual name of the class when describing it.

```suggestion
      it 'raises ActiveRecord::RecordNotFound exception' do
```